CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# Contributing to Distributed Ledger
```

Comment:
Plz update title. And yes this means that this file in sawtooth-core is really old. :man_facepalming: .